src/app/[lng]/swap/sitemap.ts

```diff
@@ -0,0 +1,67 @@
+import type { MetadataRoute } from 'next';
+import { getChainsQuery } from '@/hooks/useChains';
+import type { Token } from '@lifi/sdk';
+import { getChainById } from '@/utils/tokenAndChain';
+import coins from '@/utils/coins';
+import { getSiteUrl } from '@/const/urls';
+
+//Optimized function to generate ordered bridge pairs (tokens from different chains)
+const generateBridgeOrderedPairs = (tokens: Token[]) => {
```

Comment:
why do we keep this ?

Reply:
It is removed now. Instead the logic for the sitemap went into the general sitemap.ts which also handles most other entries, except for bridge pages.